source/ide/ide_methods.bas

```diff
@@ -195,9 +195,12 @@ FUNCTION ide2 (ignore)
 
         IF idecustomfont THEN
             idecustomfonthandle = _LOADFONT(idecustomfontfile$, idecustomfontheight, "MONOSPACE")
-            IF idecustomfonthandle = -1 THEN
-                'failed! - revert to default settings
+            IF idecustomfonthandle < 1 THEN
+                retval = idemessagebox("Custom font not found!", "Your desired font was not found at the specified location, or is\nof unsupported format. Reverting back to default built-in font.", "#OK")
                 idecustomfont = 0: idecustomfontfile$ = "C:\Windows\Fonts\lucon.ttf": idecustomfontheight = 21

```

Comment:
We can change
```vb
idecustomfontfile$ = "C:\Windows\Fonts\lucon.ttf"
```
to
```vb
idecustomfontfile$ = _DIR$("fonts") + "lucon.ttf"
```

That way it will point to the correct Windows directory (whichever drive / directory it is installed in).

As a bonus, we can also make this cross-platform by replacing 'lucon.ttf" with a font that is available on other platforms.
```vb
IF INSTR(_OS$, "[MACOSX]") THEN
    idecustomfontfile$ = _DIR$("fonts") + "Courier New.ttf"
ELSEIF INSTR(_OS$, "[LINUX]") THEN
    idecustomfontfile$ = _DIR$("fonts") + "truetype/liberation/LiberationMono-Regular.ttf"
ELSE
    idecustomfontfile$ = _DIR$("fonts") + "lucon.ttf"
END IF
```